peribolos/knative-sandbox.yaml

```diff
@@ -1132,6 +1132,7 @@ orgs:
         members:
         - maximilien
         - zhanggbj
+        - nader-ziada
```

Comment:
/assign @maximilien @zhanggbj

Reply:
my goal with being added to the kperf repo is to make it easier to get my contributions merged in as well as working on getting more folks contributing as well

Reply:
Why not be a reviewer first then we can add you as committer when you have some contributions merged.

Super happy to have you on board BTW.

Reply:
there was no kperf reviewers group to add my self to, but happy to remove myself from this for now until I have more PRs merged in. 
Already have 2 merged in and 1 in progress

Reply:
but I please ask the maintainers of the kperf repo to at least merge in dependencies PRs and such so it doesn't block progress